ksqldb-rocksdb-config-setter/pom.xml

```diff
@@ -28,6 +28,7 @@
     <artifactId>ksqldb-rocksdb-config-setter</artifactId>
     <name>ksqlDB RocksDB Config Setter</name>
     <packaging>jar</packaging>
+    <properties><mockito.version>3.8.0</mockito.version></properties>
```

Comment:
I'm confused as to why pinning this here is necessary when we're pinning it in the parent pom already.

Reply:
It's not, bad copy paste!